WordPress/src/main/java/org/wordpress/android/ui/reader/repository/usecases/GetDiscoverCardsUseCase.kt

```diff
@@ -63,6 +66,20 @@ class GetDiscoverCardsUseCase @Inject constructor(
                             ReaderConstants.JSON_CARD_RECOMMENDED_BLOGS -> {
                                 cardJson?.let {
                                     val recommendedBlogs = parseDiscoverCardsJsonUseCase.parseRecommendedBlogsCard(it)
+                                            .map { blog ->
+                                                RecommendedBlog(
+                                                        blogId = blog.blogId,
+                                                        url = blog.url,
+                                                        name = blog.name,
+                                                        iconUrl = blog.iconUrl,
+                                                        feedId = blog.feedId,
+                                                        description = blog.description,
+                                                        isFollowed = readerBlogTableWrapper.isSiteFollowed(
```

Comment:
This is the place where combining the info from two sources happens mentioned in the comment about new model